src/Core/SettingsChangesHistory.cpp

```diff
@@ -57,6 +57,446 @@ String ClickHouseVersion::toString() const
+            {"normalize_function_names", false, true, "Normalize function names to their canonical names, this was needed for projection query routing"}
+        }
+    },
+    {"21.2",
+        {
+            {"enable_global_with_statement", false, true, "Propagate WITH statements to UNION queries and all subqueries by default"}
+        }
+    },
+    {"21.1",
+        {
+            {"insert_quorum_parallel", false, true, "Use parallel quorum inserts by default. It is significantly more convenient to use than sequential quorum inserts"},
+            {"input_format_null_as_default", false, true, "Allow to insert NULL as default for input formats by default"},
+            {"optimize_on_insert", false, true, "Enable data optimization on INSERT by default for better user experience"},
+            {"use_compact_format_in_distributed_parts_names", false, true, "Use compact format for async INSERT into Distributed tables by default"}
+        }
+    },
+    {"20.10",
+        {
+            {"format_regexp_escaping_rule", "Escaped", "Raw", "Use Raw as default escaping rule for Regexp format to male the behaviour more like to what users expect"}
+        }
+    },
+    {"20.7",
+        {
+            {"show_table_uuid_in_table_create_query_if_not_nil", true, false, "Stop showing  UID of the table in its CREATE query for Engine=Atomic"}
+        }
+    },
+    {"20.5",
+        {
+            {"input_format_with_names_use_header", false, true, "Enable using header with names for formats with WithNames/WithNamesAndTypes suffixes"},
+            {"allow_suspicious_codecs", true, false, "Don't allow to specify meaningless compression codecs"}
+        }
+    },
+    {"20.4",
+        {
+            {"validate_polygons", false, true, "Throw exception if polygon is invalid in function pointInPolygon by default instead of returning possibly wrong results"}
+        }
+    },
+    {"19.18",
+        {
+            {"enable_scalar_subquery_optimization", false, true, "Prevent scalar subqueries from (de)serializing large scalar values and possibly avoid running the same subquery more than once"}
+        }
+    },
+    {"19.14",
+        {
+            {"any_join_distinct_right_table_keys", true, false, "Disable ANY RIGHT and ANY FULL JOINs by default to avoid inconsistency"}
+        }
+    },
+    {"19.12",
+        {
+            {"input_format_defaults_for_omitted_fields", false, true, "Enable calculation of complex default expressions for omitted fields for some input formats, because it should be the expected behaviour"}
+        }
+    },
+    {"19.5",
+        {
+            {"max_partitions_per_insert_block", 0, 100, "Add a limit for the number of partitions in one block"}
+        }
+    },
+    {"18.12.17",
+        {
+            {"enable_optimize_predicate_expression", 0, 1, "Optimize predicates to subqueries by default"}
+        }
+    },
```

Comment:
Tihs doesn't look good, seems like everything is duplicated.

Reply:
Seems like last merge with master went wrong

Reply:
Yep, my bad. We have `merge=union` setup and it merged cleanly locally, which was odd, so I looked into the changes and applied them in my file, without noticing the old ones were kept too. Fixing it